src/main/kotlin/oop/Command/ResponsabilityChain/PasswordProcessor.kt

```diff
@@ -0,0 +1,14 @@
+package oop.Command.ResponsabilityChain
+
+class PasswordProcessor(val next : MessageProcessor? = null) : MessageProcessor {
```

Comment:
Why are you operate with nulls? Can you rethink about this pattern and have some refractors? Or is all that you can do?